### ISSUE TITLE
Happy-Search-v1-4-7b

### ISSUE THREAD

**Issue by dinosw** [Zoltanar/Happy-Search#2]
Hi

I am getting errors when trying to do virtually anything in this version.
There have been several different errors.
Even with a clean install, I have to be very lucky to even open the program.
The most I can say, in regards to the errors, are that they have all been connected to the sqlite database.
[Screenshot](https://goo.gl/prMUew)

However, 1.4.5 works perfectly fine.
I have tried both overwriting previous install with 1.4.7 (as I normally do), and doing a fresh install thrice.

Kind Regards
Steven Windfeld

**Comment by Zoltanar** [comment]
Can you submit the log file in Stored Data (message.log)?

And what's stranger is that you say sometimes the program opens but not always, it shouldn't have such random behaviour. Do you perhaps have the database file open on a different program, such as DB Browser for SQLite?

Edit:
Actually I think I know what the issue is and I'm submitting a fix soon.

**Comment by dinosw** [comment]
I have not opened the database with any program, nor have I had different versions of the program running, at the same time.
I have tried a few reinstalls again.
I can not open the program at all (the error in my first post), after I have updated my lists, data etc. after a new install.
If I overwrite a previous install, or attempt to open the program after having updated the above mentioned, then I am met with the problem described in my first post.

Whit a completely clean install, I get an error message, it can be on updating my list, or when attempting to edit something regarding my list (if I can get that far); why I am not always getting the error upon updating my list, I sadly cannot say, but that is the error I have gotten on almost all attempts.
Here is a [Screenshot](https://goo.gl/DAEFEu) and [Error Message - Pastebin](https://goo.gl/3rzw6M)
I sadly have deleted the other problematic installs, but here is the [message.log](https://goo.gl/rJ6JLC), from my most recent clean install (with the error connected to the above Screenshot and Error Message.

I appreciate your hard work on this project, and hope the error, might be an easy one to fix, and that the info I have provided may be of some help.

PS. Sorry for the vague issue report in my first post.
PPS. I just noticed something, I have a backup of my 1.4.5 version, and the following file is alot bigger compared to the 1.4.7b version.
1.4.7b:- Happy-Search-Local-DB.sqlite = 56kb
1.4.5:-   Happy-Search-Local-DB.sqlite = 1.09mb
Thought it might be of some use to mention
